export const returnError = () => ({
  data: {
    ok: false,
    message: 'Erro interno do sistema!',
  },
});
